test/hdfs_test.py

```diff
@@ -272,7 +274,13 @@ def test_with_write_success(self):
 
 
 @attr('minicluster')
-class HdfsTargetTests(MiniClusterTestCase):
+class HdfsTargetTests(MiniClusterTestCase, BaseTargetTestCase):
```

Comment:
*Tarrasch stares at the ID*. Ok, seems like we've had almost 200 PRs in 50 days. Nice :)